src/graph/backend/dnnl/kernels/sdp_decomp_config.cpp

```diff
@@ -25,10 +29,11 @@ bool sdp_decomp_config_t::initial_check(const std::shared_ptr<subgraph_t> &sg,
         const std::vector<logical_tensor_t> &inputs) {
     // The order of input logical tensors in inputs is not certain, we need
     // to record the input offset in a certain order of ops.
-    auto op_status = record_input_offset(sg, inputs);
-    if (op_status != status::success) return false;
+    VCHECK_SDP_DECOMP(record_input_offset(sg, inputs) == status::success, false,
+            "Failed to record input offset");
```

Comment:
Any specific consideration to wrap the `record_input_offset()` function into the macro? I feel the below style is easier to read and to step into the function when debug.

```
auto op_status = record_input_offset(sg, inputs);
VCHECK_SDP_DECOMP(op_status == status::success, false,
            "Failed to record input offset");
```